codersdk/deployment.go

```diff
@@ -1265,6 +1306,29 @@ when required by your organization's security policy.`,
 			Group:         &deploymentGroupConfig,
 			Value:         &c.Config,
 		},
+		{
+			Name:        "CLI SSH Deployment Name",
+			Description: "The CLI SSH deployment name is the used in the Hostname of the ssh config.",
+			Flag:        "cli-ssh-deployment-name",
+			Env:         "CLI_SSH_DEPLOYMENT_NAME",
```

Comment:
We should probably prefix this with `CODER_` just so it aligns with our other options.

Reply:
Maybe `CODER_SSH_HOSTNAME_PREFIX` instead?

Reply:
And the default would be `coder.` (note the dot, which I think is good for people to have configurable)

Reply:
The `CODER` prefix is automatic.

You think the `.` should be included? I can do that.